### DISCUSSION TITLE
A typo found on Page 136 first paragraph

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#31] [Errata]
On page 136 the first paragraph... I think the word stronly should be the word **_strongly_**? So, the sentence makes more sense from my point of view, or am I wrong? I am not a native speaker of English.
So the beginning of the sentence would be look like this for my opinion...

> Thus, `TCollection `is not strongly typed to one particular type of columns.

**Comment by holgerflick** [comment]
Fixed. Thank you!
